docs/source/app_developers_guide/aws.rst

```diff
@@ -73,13 +73,13 @@ to log in to the instance you just launched. Use the username
 
 Once you've successfully logged in, continue to the next section.
 
-Using Sawtooth with the CLI Commands
-====================================
+Using Sawtooth Commands
```

Comment:
I prefer: "Using Sawtooth Command Line Tools"
as `intkey` is a separate CLI than `sawtooth`
Kind of a nit though.

Reply:
The CLI / command / tool terminology is nonstandard and inconsistent across the Sawtooth doc set. I am making the process of making it consistent as I edit each section.  This change matches an earlier change to the --help text and the CLI Command Reference.  

FYI, I am following the model of Red Hat, Amazon AWS, and other docs. In brief, "CLI" is the umbrella term for the entire set of Sawtooth commands; "command" is the term for the first item you type on a command line, and "subcommand" is the 2nd or 3rd-level item (before options and arguments) for a command, as in "sawtooth batch" or "sawnet peers list".